VideoLocker/res/layout/panel_custom_action_bar.xml

```diff
@@ -18,13 +18,13 @@
         android:layout_marginLeft="11dp"
         android:layout_marginStart="11dp"
         android:orientation="vertical"
+        android:contentDescription="@string/cancel_btn"
```

Comment:
If some UI element looks and behaves like a button, it should have the suffix _button_ after its name. This is something that I've noticed in the PlayStore app quite frequently, and if we use a native Button widget, say, a button with text _Login_, it will always be read as _Login Button_.
And reason I used the "I cancel button" text, is because, iOS is using the same.